src/lib/components/lines-and-dots/svg/event-detail-row.svelte

```diff
@@ -52,6 +55,16 @@
   $: textStartingY = eventY + 1.5 * fontSizeRatio;
 </script>
 
+{#if active}
+  <foreignObject {x} y={y - fontSizeRatio} {width} height={fontSizeRatio}>
+    <button
+      class="flex items-center gap-0.5 rounded-t bg-white px-2 text-sm text-black"
```

Comment:
NIT
```suggestion
      class="flex items-center gap-0.5 rounded-t bg-white pl-1.5 text-sm text-black"
```

| Before | After |
|--|--|
|![Screenshot 2024-04-19 at 2 27 42 PM](https://github.com/temporalio/ui/assets/15069288/4202485f-d9cd-42db-bed3-e07446eb6f04)|![Screenshot 2024-04-19 at 2 29 50 PM](https://github.com/temporalio/ui/assets/15069288/bc6a4de0-a6af-4633-8349-b8dfecf75329)|